### ISSUE TITLE
[BUG]: SIMD.shuffle on 64 bytes does not compile and crashes

### ISSUE THREAD

**Issue by HJLebbink** [modularml/mojo#1505]
### Bug description

```python
var data1 = SIMD[DType.uint8, 64]()
let data2 = data1.shuffle[p[0], p[1], ..., p[63]]()
```

This shuffle should compile to a [`VPERMB`](https://github.com/HJLebbink/asm-dude/wiki/VPERMB), instead the compiler crashes. Im not aware of a way to circumvent the crash.

```
mojo crash1.mojo
/__w/modular/modular/Kernels/mojo/builtin/_startup.mojo:76:1: error: no viable expansions found
/__w/modular/modular/Kernels/mojo/builtin/_startup.mojo:76:1: note:   call expansion failed - no concrete specializations
/__w/modular/modular/Kernels/mojo/builtin/_startup.mojo:11:1: note:     no viable expansions found
/__w/modular/modular/Kernels/mojo/builtin/_startup.mojo:23:14: note:       call expansion failed - no concrete specializations
/mnt/c/Source/Github/sort-networks-mojo/crash1.mojo:11:1: note:         no viable expansions found
fn main():
^
/mnt/c/Source/Github/sort-networks-mojo/crash1.mojo:86:6: note:           call expansion failed - no concrete specializations
    ]()
     ^
/__w/modular/modular/Kernels/mojo/utils/list.mojo:361:1: note:             no viable expansions found
/__w/modular/modular/Kernels/mojo/utils/list.mojo:568:26: note:               constraint failed: param assertion failed
mojo: error: failed to run the pass manager
```

### Steps to reproduce

```python
from random import random_ui64

fn gen_perm() -> StaticIntTuple[64]:
    var result = StaticIntTuple[64]()
    for i in range(64):
        result[i] = i
    return result


fn main():
    alias p = gen_perm()
    print("perm " + str(p))

    # generate random data to prevent that everything gets simplified
    var data1 = SIMD[DType.uint8, 64]()
    for i in range(64):
        data1[i] = random_ui64(0, 100).cast[DType.uint8]()
    print(data1)

    let data2 = data1.shuffle[
        p[0],
        p[1],
        p[2],
        p[3],
        p[4],
        p[5],
        p[6],
        p[7],
        p[8],
        p[9],
        p[10],
        p[11],
        p[12],
        p[13],
        p[14],
        p[15],
        p[16],
        p[17],
        p[18],
        p[19],
        p[20],
        p[21],
        p[22],
        p[23],
        p[24],
        p[25],
        p[26],
        p[27],
        p[28],
        p[29],
        p[30],
        p[31],
        p[32],
        p[33],
        p[34],
        p[35],
        p[36],
        p[37],
        p[38],
        p[39],
        p[40],
        p[41],
        p[42],
        p[43],
        p[44],
        p[45],
        p[46],
        p[47],
        p[48],
        p[49],
        p[50],
        p[51],
        p[52],
        p[53],
        p[54],
        p[55],
        p[56],
        p[57],
        p[58],
        p[59],
        p[60],
        p[61],
        p[62],
        p[63],
    ]()

    print(data2)
```

### System information

```shell
$ wsl --version
WSL version: 2.0.9.0
Kernel version: 5.15.133.1-1
WSLg version: 1.0.59
MSRDC version: 1.2.4677
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22631.2715

$ mojo --version
mojo 0.6.0 (d55c0025)

$ modular --version
modular 0.2.2 (95d42445)
```

**Comment by HJLebbink** [comment]
Hi @abduld Thanx for the swift fixes! Any estimate on the next release (or an unofficial version that I could use), This bug stall my project https://github.com/HJLebbink/sort-networks-mojo

**Comment by akirchhoff-modular** [comment]
> Any estimate on the next release (or an unofficial version that I could use), This bug stall my project https://github.com/HJLebbink/sort-networks-mojo

Our normal release schedule may have disruptions due to the holidays. I would expect a release containing this bug fix to be available by mid-to-late January.

**Comment by HJLebbink** [comment]
Update: not fixed in mojo 0.6.3 (f58f8b94)

@akirchhoff-modular Did you had a version in mind when the fix will be published?

**Comment by akirchhoff-modular** [comment]
@HJLebbink I believe the fixes should have gone out in 0.7.0, released today. Please try with 0.7.0. 🙂

**Comment by HJLebbink** [comment]
Yes, fixed, thanx!!
